ganttproject/build.gradle

```diff
@@ -8,6 +8,9 @@ dependencies {
   compile fileTree(dir: project.ext.libDir, include: ['*.jar'])
   compile project(':..:biz.ganttproject.core')
   compile files('data/resources')
+  compile 'com.google.api-client:google-api-client:1.22.0'
```

Comment:
There is a problem with `mavenDeps`.
Google libs has dependencies from `guava` and fetch `guava` lib with itself, that brokes dependencies for some files, because `guava` lib is locating in `core` package. The decision of problem is deleting `guava` lib manually, but how can I do it automatically?